src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -1737,12 +1737,27 @@ tx_gateway_frontend::reabort_tm_tx(
         auto prs = co_await when_all_succeed(pfs.begin(), pfs.end());
         auto grs = co_await when_all_succeed(gfs.begin(), gfs.end());
         auto ok = true;
+        auto fail = false;
         for (const auto& r : prs) {
-            if (r.ec != tx_errc::none) {
+            if (r.ec == tx_errc::none) {
+                // intentionally empty
```

Comment:
i do not understand this intentionally empty `if` blocks can this be: 

```c++
if(r.ec == == tx_errc::request_rejected){ 

} else if(r.ec != tx_errc::none) {

}
```


```

Reply:
sleep deprivation :) I fixed it